profile/README.md

```diff
@@ -8,7 +8,7 @@ As the leader in the creator marketing space, #paid offers end-to-end workflow m
 
 ### 💻 Our tech stack
 
-We use a range of technologies to get the job done: JavaScript and Node.js coupled with Fastify, Webpack, React, Redux, and other libraries to provide a modern, easy-to-use **Javascript** toolchain. A RESTful API back-end powers our platform services.
+We use a range of technologies to get the job done: JavaScript and Node.js coupled with Fastify, Webpack, React, Redux, Ionic, and other libraries to provide a modern, easy-to-use **Javascript** toolchain. A RESTful API back-end powers our platform services.
```

Comment:
Do we use webpack and redux explicitly?

Reply:
Webpack is part of `lib-design-system` https://github.com/hashtagpaid/lib-design-system/blob/main/package.json#L89

Are you suggesting we remove webpack and redux from the readme?

Reply:
https://github.com/hashtagpaid/lib-design-system/blob/main/package.json#L33

I thought we use rollup for release builds. 

I'm just thinking that these dependencies maybe required via another dependency like CRA. But we don't use it directly. (If that makes sense.)

This comment is more of a question/clarification for me

Reply:
I think we used webpack and redux at some point (and we might still do), but I don't think they add much value here and I'd say we could remove them. Worth mentioning tailwind, too?

Reply:
+1 tailwind, ionic, capacitor

Reply:
Appreciate the feedback. I've updated the PR!